pkg/controllers/podgroup_test.go

```diff
@@ -281,7 +281,6 @@ func setUp(ctx context.Context,
 	objs := []runtime.Object{pg}
 	if len(podNames) != 0 {
 		ps := makePods(podNames, pgName, podPhase, podOwnerReference)
-		// s.AddKnownTypes(clientgoscheme.SchemeGroupVersion, ps)
```

Comment:
remove this.

Reply:
This comment had been removed.

Reply:
It's still there :)

Reply:
Am I misunderstanding you? This line had been remove in this pr already. 
Did you mean remove this line or uncomment this line?

Reply:
Ah my bad... somehow I always perceived this line was newly introduced...